lib/gush/worker.rb

```diff
@@ -75,6 +81,8 @@ def enqueue_outgoing_jobs
           end
         end
       end
+    rescue RedisMutex::LockError
+      Worker.perform_later(*[workflow_id, job.name])
```

Comment:
Can it be simply `Worker.perform_later(workflow_id, job.name)`? Wrap the params in an array then flattening is kind of double work

Reply:
Maybe we should delay the next job's execution by a second or two? I we keep hammering while the lock is active this might need multiple job runs to finish .

Reply:
Thank you @pokonski, @phthhieu. I updated by your comments.